### ISSUE TITLE
Name Copilot Chat, Copilot Chat instead of just Chat on VSCode settings

### ISSUE THREAD

**Issue by evandrocoan** [microsoft/vscode-copilot-release#339]
It would be nice if the Copilot Chat would be named Copilot Chat instead of just Chat.

I only found this extension had settings after finding this issue (https://github.com/microsoft/vscode-copilot-release/issues/13). After searching for Copilot, on VSCode settings, I did not find the Copilot Chat settings. 

![image](https://github.com/microsoft/vscode-copilot-release/assets/5332158/87e0b105-1fae-4830-86d8-8e7dcffc8373)

I had no idea that after installing an extension named Copilot Chat, its name could be only Chat on settings:

![image](https://github.com/microsoft/vscode-copilot-release/assets/5332158/93503cb6-34ba-45f8-a405-356ab831e960)

![image](https://github.com/microsoft/vscode-copilot-release/assets/5332158/da6c8241-d806-4c08-b5c2-aac983098cc4)

**Comment by jrieken** [comment]
I agree that this is confusing but there are two parts here: the chat feature is built into VS Code and is extension agnostic. It comes with settings, like font sizes etc. Now, extensions make the chat feature/UI shine and fill it with live, such as copilot chat. These extensions can also have settings. This isn't a novel issue, e.g we have generic code lens settings and specific TypeScript code GitLens code lens settings, and I believe it should boil down to how we group and discover these settings in the UI, not strictly how we name them.

**Comment by roblourens** [comment]
Agreed, and fyi @jrieken we discussed this briefly in Slack while you were out https://vscodeteam.slack.com/archives/C05BHT75R6G/p1690420127584859

**Comment by isidorn** [comment]
+100 here, also discussed this with @bpasero 
Also it would align better with the existing "Copilot Edits".

**Comment by rzhao271** [comment]
Moving from Backlog Candidates to under-discussion considering team input above.

Rob suggests in https://github.com/microsoft/vscode-copilot-release/issues/339#issuecomment-1668862791 to not add Copilot onto the core settings but to improve the search algorithm to detect related settings. I currently don't have synonyms/related settings search in the Settings editor, but would love to add in such a feature once I have a better idea how to implement it.

@isidorn I notice you wrote +100 in https://github.com/microsoft/vscode-copilot-release/issues/339#issuecomment-2532657854. Could you elaborate on your thoughts?

**Comment by isidorn** [comment]
@rzhao271 my thinking:
1) "chat" is too generic (as the user shared in their comment)
2) "copilot" is actually becoming a generic term, and not only married to the MS brand (you can see how many VS Code extensions use the term copilot in tag/description)
3) There is an existing view that we call "Copilot Edits"

However we do want to bring the Chat and Edits view together into one view. And once that happens we can finalise the name. The view could be Copilot, and then it has multiple modes, Chat and Edits that user can toggle between.
